roles/pacman/files/pacman-pkgs-core

```diff
@@ -36,7 +36,6 @@ facter
 xf86-video-vesa
 nvidia
 nvidia-utils
-lib32-nvidia-utils
```

Comment:
Maybe this had temporarily been removed when you installed your OS, but it is currently a arch package: https://www.archlinux.org/packages/multilib/x86_64/lib32-nvidia-utils/